### PR TITLE
[release/1.1.0] Mark packages as stable for 1.1.3 release

### PR DESCRIPTION
**Pull request by weshaggard**
@steveharter @eerhardt @ericstj

### CODE REVIEW

**Comment by eerhardt** [comment]
I assume the error

`12:49:09 error: Unable to resolve 'runtime.native.System.IO.Compression (>= 4.3.1)' for '.NETCoreApp,Version=v1.1'.`

is expected since these packages aren't available publicly yet, right?

File: build_projects/shared-build-targets-utils/Utils/BuildVersion.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ public class BuildVersion : Version
     {
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
-        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}-00";
+        public string NetCoreAppVersion => $"{ProductionVersion}" + (HostVersion.EnsureStableVersion ? "" : $"-{VersionSuffix}-00");
```
Comment (eerhardt) [inline build_projects/shared-build-targets-utils/Utils/BuildVersion.cs]: I'm surprised changes like this are still needed after 6 service releases.

**Comment by weshaggard** [comment]
>is expected since these packages aren't available publicly yet, right?

Correct.

File: build_projects/shared-build-targets-utils/Utils/BuildVersion.cs (hunk 1 above)
Comment (weshaggard) [inline build_projects/shared-build-targets-utils/Utils/BuildVersion.cs]: Yeah I'm just trying to make it a little easier for the next time.

**Pull request merged**